lib/better_words/words/worst_word.ex

```diff
@@ -0,0 +1,18 @@
+defmodule BetterWords.Words.WorstWord do
+  use BetterWords.Schema
```

Comment:
Normally you would write `use Ecto.Schema`. This like covers that but also brings in the other boilerplate settings that we want consistently across all of our schema files.

Reply:
This file was generated but I followed up and changed out this line and removed the (now) duplicate code.